platform/windows/include/unistd.h

```diff
@@ -0,0 +1,58 @@
+#ifndef _UNISTD_H
```

Comment:
I would make clear which files are just platform-specific implementations for MapLibre and which are replacements for system headers.

Also I'm not sure if I like we're hacking system-like headers. How many times do we actually need functions from this file?

Reply:
I can check which functions from this code are used. All I can say by now is some files are including unistd.h. We can trace back these includes and make some deeper changes than this hack.